introduction.md

```diff
@@ -1,3 +1,11 @@
 I'm not good with words... so, it's just a file to atend the lesson.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?